DependencyInjection/Security/Factory/OAuthFactory.php

```diff
@@ -36,7 +37,11 @@ public function create(ContainerBuilder $container, $id, $config, $userProvider,
             ;
 
         $listenerId = 'security.authentication.listener.fos_oauth_server.'.$id;
-        $container->setDefinition($listenerId, new DefinitionDecorator('fos_oauth_server.security.authentication.listener'));
+        if (version_compare(Kernel::VERSION, '2.6', '>=')) {
```

Comment:
please use feature detection rather than version detection. When using standalone components, the HttpKernel and the DependencyInjection components might be at different versions